src/main/java/me/botsko/prism/Prism.java

```diff
@@ -52,7 +52,7 @@
     /**
      * Connection Pool
      */
-    private static DataSource pool = new DataSource();
+    private static volatile DataSource pool = new DataSource();
```

Comment:
`volatile` means that the variable is consistent across threads as it isn't cached by the CPU.